core/internal/cltest/cltest.go

```diff
@@ -445,9 +443,21 @@ func NewApplicationWithConfig(t testing.TB, cfg chainlink.GeneralConfig, flagsAn
 				t.Fatal(err)
 			}
 		}
+
+		opts := solana.ChainSetOpts{
```

Comment:
very hard to differentiate intention btw solana and solana2 pkg names. had to read multiple times to find solana2 and i can't tell what makes it different from solana